apl/tabular-operators/project-reorder-operator.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: 'project-reorder'
+description: 'This page explains how to reorder specified fields in the output.'
+tags: 
+    ['axiom documentation', 'documentation', 'axiom', 'tabular operators', 'project-reorder']
+---
+
+Reorders specified fields in the output while keeping the original order of unspecified fields.
+
+## Syntax
+
+```kusto
+| project-reorder FieldName1, FieldName2, ... [asc|desc]
+```
+
+## Arguments
+
+| **name**              | **type**     | **description**                                      |
+| --------------------- | ------------ | ------------------------------------------------------ |
+| Field Name            | **string**     | The name of the field to be reordered in the output. |
+| [asc|desc]           | **string** |  Optional. Specifies the sort order for the reordered fields. |
```

Comment:
We support the `granny-asc` and `granny-desc` as well

Reply:
just for some extra description on grannies 
> granny-asc or granny-desc orders by ascending or descending, respectively, while secondarily sorting by the next numeric value. For example, a100 comes before a20 when granny-asc is specified.

that's from the kql docs

Reply:
@gordallott makes sense, added. all good now